politeiad/backendv2/tstorebe/store/mysql/mysql.go

```diff
@@ -23,7 +23,7 @@ import (
 
 const (
 	// Database options
-	connTimeout     = 1 * time.Minute
+	connTimeout     = 3 * time.Minute
```

Comment:
This might be machine specific but I had to bump it in order to avoid a mysql `context timeout` error when
running `politeiad --fsck` after importing some legacy proposals.